src/http/article.controller.js

```diff
@@ -19,9 +19,9 @@ class ArticleController extends BaseController {
                 throw new HttpException(400, result.info);
             }
 
-            return res.status(200).send(`Article [${result._id}] created successfully!`);
+            return res.redirect(routes.articleRead({ id: result.data._id.toString() }));
         } catch (error) {
-            return this.reportBadData(error, req, res).redirect(routes.articleList());
+            return this.reportBadData(error, req, res).redirect(routes.articleNew());
```

Comment:
We need to send the content of the article back to the article editor so the user doesn't lose it in case of a validation error. We can achieve this through the use of flash messages just like we do with validation errors.